dome9/data_source_dome9_awp_aws_onboarding.go

```diff
@@ -0,0 +1,141 @@
+package dome9
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"log"
+)
+
+func dataSourceAwpAwsOnboarding() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceAwpAwsOnboardingRead,
+
+		Schema: map[string]*schema.Schema{
+			"id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"centralized_cloud_account_id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"scan_mode": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"agentless_account_settings": {
+				Type:     schema.TypeSet,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"disabled_regions": {
+							Type:     schema.TypeList,
+							Computed: true,
+							Elem: &schema.Schema{
+								Type: schema.TypeString,
+							},
+						},
+						"scan_machine_interval_in_hours": {
+							Type:     schema.TypeInt,
+							Computed: true,
+						},
+						"max_concurrence_scans_per_region": {
```

Comment:
can we fix the typo and replace "max_concurre**nce**_scans" either with "max_concurre**ncy**_scans" or "max_concurre**nt**_scans" everywhere?

Reply:
@bzemms we need also to change it in the module

Reply:
@bzemms will do it also on the module

Reply:
@bzemms don't have permissions to update the module